main.py

```diff
@@ -0,0 +1,48 @@
+from gym_jadx.envs.jadx_env import JadxEnv
+import pygame
+import numpy as np
+import time
+import cv2
+
+env = JadxEnv()
```

Comment:
Because the environment is implemented as a gym environment I think initializing it like a gym environment would be better:

```suggestion
env = gym.make('jadx-v0')
```

Reply:
Yes, changed it in the last commit